### PR TITLE
Add local requirements file and a `make install`

### PR DESCRIPTION
**Pull request by adam-incuna**
I noticed that `djangorestframework` wasn't being installed in `test_requirements.txt` (since Travis needs to vary it) so I added a new requirements file.

@incuna/backend quick review please? :)

### CODE REVIEW

**Comment by meshy** [comment]
Please don't do this. I have a real problem with having multiple requirements files (and even with the `test_requirements.py`).

I would rather that you rename the existing file to be `requirements.txt`, and just override the installation in `.travis.yml` where appropriate.

**Comment by adam-incuna** [comment]
@incuna/backend updated!

File: Makefile [hunk 1]
```diff
@@ -10,3 +10,7 @@ release:
 test:
 	@py.test --cov-report term-missing --cov incuna_test_utils tests
 	@flake8 .
+
+install:
+	pip install -r test_requirements.txt
+	pip install -r test_local_requirements.txt
```
Comment (KevinEtienne) [inline Makefile]: I think if you rename `test_requirements.txt` to `requirements.txt` you need to update this

Comment (meshy) [inline Makefile]: This still needs updating

Comment (adam-incuna) [inline Makefile]: Oh woops

Comment (adam-incuna) [inline Makefile]: Done, cheers!

**Comment by adam-incuna** [comment]
Fix'd.

**Comment by adam-incuna** [comment]
Thanks :) will deploy presently!

**Pull request merged**